packages/react-percy/src/cli.js

```diff
@@ -62,9 +69,20 @@ export function run(argv, rootDir) {
       process.on('exit', () => process.exit(0));
     })
     .catch(err => {
-      // eslint-disable-next-line no-console
-      console.log(chalk.bold.red(err.stack || err.message || err));
-
-      process.on('exit', () => process.exit(1));
+      if (err.name === 'StatusCodeError') {
+        if (err.statusCode >= 500) {
+          // eslint-disable-next-line no-console
+          console.log(chalk.bold.yellow('Percy Error: %s'), formatError(err));
```

Comment:
This is where we handle a Percy outage. Thoughts on wording here? Right now it just prints the percy error in yellow and exits with code 0.

Reply:
This is what we do in ember-percy, may be close: https://github.com/percy/ember-percy/blob/master/index.js#L97

But in general this is probably fine, perhaps more explanation.